src/rbr/RBR.java

```diff
@@ -343,8 +343,7 @@ private boolean mergeUnitary(Vertex router) {
 
 	private void addRoutingPath(Vertex v, Set<Character> ip, Vertex dst, Set<Character> op) {

```

Comment:
Could you please add a Ref commit to get ride of this method and replace the calls made to it by "routingPathForVertex.get(v).add(new RoutingOption(ip, dst, op));"?